v2/thirdparty/serverless/with-aws-lambda/backend-config.mdx

```diff
@@ -4,22 +4,68 @@ title: 2. Backend config
 hide_title: true
 ---
 
+<!-- COPY SECTION -->
+<!-- ./emailpassword/serverless/with-aws-lambda/backend-config.mdx -->
+<!-- 1 -->
+
 import AppInfoForm from "/src/components/appInfoForm"
 import CoreInjector from "/src/components/coreInjector"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
 
 # 2. Backend config
 
-## 1) Install supertokens package
+## 1) Create Lambda layer with required libraries
+
+<BackendSDKTabs disableGolang>
+<TabItem value="nodejs">
+
 ```bash
-npm i supertokens-node
+mkdir lambda && cd lambda
+
+npm i -s supertokens-node @middy/core @middy/http-cors
+mdkir nodejs && cp -r node_modules nodejs
+zip -r supertokens-node.zip nodejs/
 ```
 
-## 2) Create a configuration file (`config.js`)
-- Create a `config.js` file in the root directory of your project.
-- An example of this file can be found [here](https://github.com/supertokens/supertokens-auth-react/blob/master/examples/with-aws-lambda/backend/config.js).
+</TabItem>
+<TabItem value="python">
+
+```bash
+mkdir lambda && cd lambda
+
+pip install --target ./python fastapi uvicorn mangum nest_asyncio supertokens-python
+cd python && rm -r *dist-info **/__pycache__ && cd ..
+zip -r supertokens-python.zip python/
+```
+
+</TabItem>
+</BackendSDKTabs>
+
+## 2) Upload SuperTokens lambda layer
+- Open AWS Lambda dashboard and click on layers:
+<img src="https://user-images.githubusercontent.com/20884782/190074706-3d1b6420-e812-48d9-aa04-8ea2d4a4c62d.png" height="300" />
```

Comment:
Change all the images to come from the static image folder in the v2 folder in the docs

Reply:
done